tests/test_other.py

```diff
@@ -173,6 +173,7 @@ def run_on_pty(self, cmd):
       os.close(master)
       os.close(slave)
 
+  @with_env_modify({'EMCC_CFLAGS': '-should -be -ignored'})
```

Comment:
Maybe add a comment here (that `-v` by itself ignores these flags which would cause an error otherwise) and/or a link to this issue?

Reply:
Added a comment